plugins/terraform-backend/src/lib/index.ts

```diff
@@ -4,6 +4,21 @@ import { formatTerraformRun } from './formatTerraformRun';
 
 export const TF_BASE_URL = 'https://app.terraform.io/api/v2';
 
+export const getLatestRunForWorkspace = async (
+  token: string,
+  workspaceId: string,
+) => {
+  const res = await axios.get<TerraformResponse<TerraformRun[]>>(
+    `${TF_BASE_URL}/workspaces/${workspaceId}/runs?include=created_by,plan&page%5B1%5D`,
```

Comment:
is the page query param correct here? how would the terraform api know if you are referring to the page number or the page size?
https://developer.hashicorp.com/terraform/cloud-docs/api-docs/run#query-parameters

Reply:
I believe the query params should be as below:
`/runs?include=created_by,plan&page[number]=1&page[size]=1`